Beginner/MultipleScheduledTasks/src/main/java/executor/Executor.java

```diff
@@ -0,0 +1,17 @@
+package executor;
+
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class Executor {
+
+    public void execute(List<Schedule> schedulers) {
+        ExecutorService executorService = Executors.newFixedThreadPool(10);
+        for(Schedule scheduler : schedulers) {
+            executorService.submit(scheduler);
+        }
+        executorService.shutdown();
```

Comment:
Won't this stop the execution?

Reply:
It will shutdown only after all the running threads finish their work. Since in our case we have an infinite loop in the run method, it won't stop the execution.